typescript/db/package.json

```diff
@@ -10,7 +10,8 @@
     "stop": "docker compose down --volumes --remove-orphans",
     "deploy": "prisma migrate deploy",
     "dev:push": "prisma db push",
-    "generate": "prisma generate"
+    "generate": "prisma generate",
+    "resolve": "prisma migrate resolve --applied  20210601170000_add_strapi"
```

Comment:
Do we want to keep this?

Reply:
Wow not at all, good catch, will remove it!

Reply:
Done in c93498eb53844378be5e7e84c6d78c34cd5699a5